client/src/components/ImageUploader.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { BsPlusSquareDotted } from 'react-icons/bs'
+import styled from 'styled-components';
+
+const UploadContainer = styled.div`
+  font-size: 64px;
+  height: 400px;
```

Comment:
Setting hard values for containers has almost always come back to bite me in the butt. It might be a good idea for us to talk about how we set container height and width values to reduce tech debt. I also said something similar to Jordan on his PR so I think we should all get together and discuss.